package.json

```
@@ -59,16 +59,16 @@
     "prettier": "2.6.2"
   },
   "dependencies": {
-    "@neume-network/message-schema": "0.3.1",
-    "@neume-network/schema": "0.5.0",
+    "@neume-network/schema": "^0.6.0",
```

```suggestion
    "@neume-network/schema": "0.6.1",
```

let's not use the caret as it is dangerous and can lead to supply chain attacks